src/Database/PostgreSQL/PQTypes/Model/Migration.hs

```diff
@@ -64,13 +64,13 @@ data MigrationAction m =
   -- Name of the table that the cursor is associated with. It has to be the same as in the
   -- cursor SQL, see the second parameter.
   --
-  -- SQL that will be used for the cursor.
+  -- SQL providing a list of primary keys from the associated table that will be used for the cursor.
   --
-  -- Function that takes a list of primary keys provided by the cursor SQL and
+  -- Modification function that takes a batch of primary keys provided by the cursor SQL and
```

Comment:
Why would you point out that this function is a `Modification` function? It's called `ModifyColumnMigration` and the docs also say that it runs an arbitrary computation... To me, stating that this function modifies something is quite excessive and makes the doc a bit annoying to read.

Reply:
👍🏽 sure it does look annoying now. I'll remove `Modification`